.github/workflows/deploy-docker.yml

```diff
@@ -88,6 +102,7 @@ jobs:
                     images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
             -
                 name: Set Version number
+                if: ${{ steps.branch-name.outputs.is_default && steps.branch-name.outputs.default_branch != '' && github.actor != 'dependabot[bot]' }}
```

Comment:
Only set the version number if its not a PR

Setting the version number will mean it pushes to the version tag, meaning a PR could overwrite the normal version tag such as `1.0.5` this stops that from happening.